ASSIGMENT/src/pages/dl.astro

```diff
@@ -0,0 +1,271 @@
+                        <input type="radio" id="gender_female" name="gender" value="female" required>
+                        <label for="gender_female">Female</label>
+                    </div>
+                </div>
+            </div>
+        </form>
+
+        <div class="button-container">
+            <button type="button" class="resetButton">Reset</button>
+            <button type="submit" class="saveButton">Save</button>
+        </div>
+    </section>
+    <div class="container">
+        <div class="item header">
+            <div>#</div>
+            <div>Document</div>
+            <div>Info</div>
+            <div>Name</div>
+            <div>Gender</div>
+            <div>DOB</div>
+            <div>Actions</div>
+        </div>
+    </div>
+    <div id="canvasContainer" style="display: none;">
+        <canvas id="dlCanvas"></canvas>
+        <button id="downloadButton" style="display: none;">Download Image</button>
+    </div>
+
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
+            const dlForm = document.getElementById("dlForm");
+            const saveButton = document.querySelector(".saveButton");
+            const resetButton = document.querySelector(".resetButton");
+            const container = document.querySelector(".container");
+            const canvasContainer = document.getElementById("canvasContainer");
+            const downloadButton = document.getElementById("downloadButton");
+            const canvas = document.getElementById('dlCanvas');
+            const context = canvas ? canvas.getContext('2d') : null;
+            let entryCount = 0;
+            let currentEditItem = null;
+
+            if (!context) {
+                console.error("Canvas context could not be obtained.");
+                return;
+            }
+
+            const handleSubmit = (event) => {
+                event.preventDefault();
+                const documentNumber = dlForm.querySelector("#documentNumber_dl").value.trim();
+                const holdingPersonName = dlForm.querySelector("#holdingPersonName_dl").value.trim();
+                const DOB = dlForm.querySelector("#DOB_dl").value.trim();
+                const gender = dlForm.querySelector('input[name="gender"]:checked').value;
+                const DOE = dlForm.querySelector("#DOE_dl").value.trim();
+
+                if (documentNumber && holdingPersonName && DOB) {
+                    if (currentEditItem) {
+                        updateEntry(currentEditItem, documentNumber, holdingPersonName, DOB, gender,DOE);
+                    } else {
+                        createNewEntry(documentNumber, holdingPersonName, DOB, gender, DOE);
+                    }
+                    resetForm();
+                } else {
+                    console.log("Please fill in all required fields.");
+                }
+            };
+
+            const createNewEntry = (documentNumber, holdingPersonName, DOB, gender, DOE) => {
```

Comment:
use object in parampeters .  no uppercase lowercase mixing
```suggestion
            const createNewEntry = ({number, name, dob, gender, doe}) => {
```

Reply:
done